dataland-e2etests/src/test/kotlin/org/dataland/e2etests/CompanyDataControllerTest.kt

```diff
@@ -4,48 +4,88 @@ import org.dataland.datalandbackend.openApiClient.api.CompanyDataControllerApi
 import org.dataland.datalandbackend.openApiClient.api.EuTaxonomyDataControllerApi
 import org.dataland.datalandbackend.openApiClient.api.MetaDataControllerApi
 import org.dataland.datalandbackend.openApiClient.model.CompanyAssociatedDataEuTaxonomyData
-import org.dataland.datalandbackend.openApiClient.model.CompanyMetaInformation
+import org.dataland.datalandbackend.openApiClient.model.CompanyInformation
 import org.dataland.datalandbackend.openApiClient.model.DataMetaInformation
-import org.dataland.datalandbackend.openApiClient.model.PostCompanyRequestBody
+import org.dataland.datalandbackend.openApiClient.model.StoredCompany
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Assertions.assertTrue
 import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+import java.time.LocalDate
 
 class CompanyDataControllerTest {
 
-    private val basePathToBackendViaProxy = "http://proxy:80/api"
-    private val metaDataControllerApi = MetaDataControllerApi(basePathToBackendViaProxy)
-    private val companyDataControllerApi = CompanyDataControllerApi(basePathToBackendViaProxy)
-    private val euTaxonomyDataControllerApi = EuTaxonomyDataControllerApi(basePathToBackendViaProxy)
+    private val basePathToDatalandProxy = "http://proxy:80/api"
+    private val metaDataControllerApi = MetaDataControllerApi(basePathToDatalandProxy)
+    private val companyDataControllerApi = CompanyDataControllerApi(basePathToDatalandProxy)
+    private val euTaxonomyDataControllerApi = EuTaxonomyDataControllerApi(basePathToDatalandProxy)
+    private val testCompanyList1 = listOf(
+        CompanyInformation(
+            companyName = "Test-Company_1",
+            headquarters = "Test-Headquarters_1",
+            industrialSector = "Test-IndustrialSector_1",
+            marketCap = BigDecimal(100),
+            reportingDateOfMarketCap = LocalDate.now()
+        ),
+        CompanyInformation(
```

Comment:
Das ist jetzt der vierte Test in dem in irgendeiner Form dummy company Daten angelegt werden, hardcoded im jeweiligen Skript. Es wäre besser das einmal zentral zu machen und die Tests entsprechend darauf zugreifen zu lassen.

Reply:
Ja finde ich gut. Ich verfrachte das mal in den `DummyDataCreator`.  Der macht das ja bereits für DummyTaxonomy Daten. Er kann es auch für Dummy Companies übernehmen.

Reply:
_Beginne fixen_